docs/src/pages/guides/testing/testing.md

```diff
@@ -97,11 +97,11 @@ import { createMount } from '@material-ui/core/test-utils';
 describe('<MyComponent />', () => {
   let mount;
 
-  before(() => {
+  beforeEach(() => {
```

Comment:
We're using `mocha` and should say that as @joshwooding suggested. 

However, if you want to migrate these tests to `jest` you should use [`beforeAll`](https://jestjs.io/docs/en/api#beforeallfn-timeout). That should have the same semantics as `before` in `mocha`.

Reply:
Should we add two examples, one with mocha, on with jest? Jest is at least x2 more popular than mocha.

Reply:
Is there a difference beyond `before` vs `beforeAll`?

Reply:
For this code we are showcasing in the documentation, no.

Reply:
Then I don't see a point in duplicating. Simple comment might be enough.

Reply:
👍

Reply:
Done.